apps/distgo/params/docker.go

```diff
@@ -15,37 +15,45 @@
 package params
 
 import (
+	"encoding/base64"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+
 	"github.com/pkg/errors"
 )
 
 type DockerDepType string
+type DockerImageType string
 
 const (
-	DockerDepSLS    DockerDepType = "sls"
-	DockerDepBin    DockerDepType = "bin"
-	DockerDepRPM    DockerDepType = "rpm"
-	DockerDepDocker DockerDepType = "docker"
+	DockerDepSLS          DockerDepType = "sls"
+	DockerDepBin          DockerDepType = "bin"
+	DockerDepRPM          DockerDepType = "rpm"
+	DockerDepDocker       DockerDepType = "docker"
+	ManifestLabel         string        = "com.palantir.sls.manifest"
```

Comment:
This variable and the two after it shouldn't have `string` type declared explicitly. Also add a newline between these and the `DockerDepType` block before it to make it clear that they're conceptually separate.